package-lock.json

```diff
@@ -1197,9 +1197,9 @@
       }
     },
     "@hathor/wallet-lib": {
-      "version": "0.40.4",
-      "resolved": "https://registry.npmjs.org/@hathor/wallet-lib/-/wallet-lib-0.40.4.tgz",
-      "integrity": "sha512-lGLENhyDEpzI/SF+I1U/oUCd4vjCcVb7B1TtpCXhsLHAmb8vv16IrEclm2JCyH0bXG4kZldkEGLepnS97K0vOw==",
+      "version": "0.41.0",
```

Comment:
lodash is not a new dependency, but the linter required that we explicitly add it to dependencies because we import it in the project.

Just updated the package-lock with the version 0.41.1, when i removed the bitcore-lib changes I must've removed this as well.